mlir/include/mlir/Dialect/SparseTensor/IR/SparseTensorAttrDefs.td

```diff
@@ -307,6 +326,12 @@ def SparseTensorEncodingAttr : SparseTensor_Attr<"SparseTensorEncoding",
     // The required bitwidth for coordinate storage.
     "unsigned":$crdWidth,
 
+    // The required explicit value.
+    "::mlir::Attribute":$explicitVal,
```

Comment:
nit: I am not sure whether there is any style guidance for this, should it (as well as other field) use underscore (i.e., `$explicit_value`)? See https://github.com/llvm/llvm-project/blob/c6e01627acf8591830ee1d211cff4d5388095f3d/mlir/include/mlir/Dialect/Tosa/IR/TosaOpBase.td#L79.

Reply:
After offline discussion, we decided to use explicitVal (same format as the other values in the same context).